module.json

```diff
@@ -1,36 +1,43 @@
 {
   "title": "Diwako's Cyberpunk Red - Core Additions",
   "description": "Adds small features to the system as a stop gap solution until similar and/or better functionality is added to the core system itself.",
-  "version": "0.1.0",
+  "version": "0.2.0",
```

Comment:
No need. this will be set via github release build pipeline.

Reply:
Coolio, fixed!